src/mumble/CMakeLists.txt

```diff
@@ -48,12 +48,14 @@ elseif(UNIX)
 		option(coreaudio "Build support for CoreAudio." ON)
 	elseif(${CMAKE_SYSTEM_NAME} STREQUAL "FreeBSD")
 		option(oss "Build support for OSS." ON)
+	elseif(${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD")
+		option(xinput2 "Build support for XI2." ON)
```

Comment:
I think it'd be better if common options were declared in a common if-block instead of duplicating them into multiple blocks.

Aka: I would add a new if like so:
```cmake
if(${CMAKE_SYSTEM_NAME} STREQUAL "OpenBSD" OR ${CMAKE_SYSTEM_NAME} STREQUAL "Linux")
	option(xinput2 "Build support for XI2." ON)
endif()
```
and remove the lines from the individual blocks.

I think that should also solve the issue with the doc-script, right?

Reply:
> I think it'd be better if common options were declared in a common if-block instead of duplicating them into multiple blocks.
> [...]
> and remove the lines from the individual blocks.

Sure, can do.
 
> I think that should also solve the issue with the doc-script, right?

Yes, I didn't do it right away so as to not ignore the "doc-script" bug.
Given this is considered out-of-scope for now, I'll just go the easy way.